article/app/views/fragments/email/emailArticleBody.scala.html

```diff
@@ -88,30 +151,23 @@ <h1>@article.trail.headline</h1>
 
     @fragments.emailMainMedia(article)
 
-    @row(Seq("padded", "author")) {
-        @article.trail.byline.map { byline =>
-            <h3 class="byline">@byline</h3>
-
-        @article.tags.contributors.headOption.map { profile =>
-            @profile.properties.twitterHandle.map { twitter =>
-                <div class="meta__twitter">
-                    <a href="https://twitter.com/@twitter" data-link-name="twitter-handle" data-component="meta-twitter-handle" class="button button--small button--secondary tone-colour">
-                        @icon("twitter-bird")
-                        <span class="contact">@@@twitter</span>
-                    </a>
-                </div>
-            }
-            @profile.properties.emailAddress.map { email =>
-                <div class="meta__email">
-                    <a href="mailto:@email" data-link-name="email-address" data-component="meta-email-address" class="button button--small button--secondary tone-colour">
-                        @icon("mail")
-                        <span class="contact">Email</span>
-                    </a>
-                </div>
+    @page.article.trail.byline.map { byline =>
+        @* Show byline if there is exactly one contributor *@
+        @* and this is the Coronavirus: the week explained *@
+        @* and the contributor has a contributor image *@
+        @if(article.tags.contributors.length == 1 &&
+            article.tags.series.exists {
+                _.id == "world/series/coronavirus-the-week-explained"
+            }) {
+            @article.tags.contributors.headOption.map { profile =>
+                @if(profile.properties.contributorLargeImagePath.isEmpty) {
```

Comment:
I wonder if it would be possible to pull this nested `if` out and move it up to the outer conditional? Only because I think flat conditionals usually win when it comes to readability. Not sure how tricky that would be in this instance, so definitely feel free to ignore :-)

Reply:
I agree. I did tinker around with this, however my Scala is not hot enough. If anyone has a suggestion on how to do this, I'd be happy to give it a go!

Reply:
Hey yeah having a quick play locally and I'll ping you. I reckon pulling out the image check to a function is probably the way to go.

Reply:
I will fix this in a follow up PR 👍